systemtest/src/main/java/io/strimzi/systemtest/utils/kafkaUtils/KafkaUserUtils.java

```diff
@@ -186,4 +191,33 @@ public static void waitForKafkaUserMappingIntoKafkaResource(String namespace, St
                 return getUserResult.contains(userName);
             });
     }
+
+    public static void modifyKafkaUserPasswordWithNewSecret(
+        String ns, String kafkaUserResourceName, String customSecretSource, String customPassword,
+        ResourceManager resourceManager, ExtensionContext extensionContext) {
```

Comment:
Why do you passing resourceManager as a method parameter? You can do `ResourceManager.getInstance()`